scripts/gas-refund-program/transactions-indexing/transaction-resolver.ts

```diff
@@ -125,13 +144,22 @@ export const getAllTXs = async ({
     });
   }
 
-  return getTransactionForContract({
-    epoch,
-    chainId,
-    startTimestamp,
-    endTimestamp,
-    contractAddress,
-  });
+  const filteredTxs = (
+    await DuneTransaction.findAll({
+      where: {
+        chainId,
+        to: contractAddress,
+        block_timestamp: {
+          [Op.gte]: startTimestamp,
+          [Op.lte]: endTimestamp,
```

Comment:
minor but I'd be strict on one of the side to prevent double spending cross epochs

Reply:
just faced a duplicated, so that is what would happen instead of double spending, will add fix